src/main/java/com/weaponcharges/WeaponChargesPlugin.java

```diff
@@ -214,15 +214,35 @@ private void enableDevMode()
 		eventBus.register(devtools);
 	}
 
+	private int lastDegradedHitsplatTick = -1000;
+
 	@Subscribe
 	public void onHitsplatApplied(HitsplatApplied e) {
-		if (e.getActor() != client.getLocalPlayer() || e.getHitsplat().getHitsplatType() != Hitsplat.HitsplatType.DAMAGE_ME) return;
-		ChargedWeapon crystalHelm = getEquippedChargedWeapon(EquipmentInventorySlot.HEAD);
-		if (crystalHelm != null) addCharges(crystalHelm, -1, false);
-		ChargedWeapon crystalBody = getEquippedChargedWeapon(EquipmentInventorySlot.BODY);
-		if (crystalBody != null) addCharges(crystalBody, -1, false);
-		ChargedWeapon crystalLegs = getEquippedChargedWeapon(EquipmentInventorySlot.LEGS);
-		if (crystalLegs != null) addCharges(crystalLegs, -1, false);
+		Hitsplat.HitsplatType hitType = e.getHitsplat().getHitsplatType();
+		ChargedWeapon helm = getEquippedChargedWeapon(EquipmentInventorySlot.HEAD);
+		if (helm == ChargedWeapon.SERPENTINE_HELM) {
+			if (hitType == Hitsplat.HitsplatType.DAMAGE_ME || hitType == Hitsplat.HitsplatType.BLOCK_ME) {
```

Comment:
Could use a comment explaining what DAMAGE_ME and BLOCK_ME and why they are appropriate and sufficient, as the javadoc is not very clear.

Reply:
idc too much though.

Reply:
Left a note explaining the splats.